doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1052 @@
+
+  * Information about :ref:`gs_supported_OS` to the :ref:`gs_recommended_versions` page.
+  * Information about :ref:`app_build_output_files` on the :ref:`app_build_system` page.
+  * Information about :ref:`gs_debugging` on the :ref:`gs_testing` page.
+    Also added links to this section in different areas of documentation.
+
+* Updated :ref:`app_memory` page with sections about Gazell and NFC.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for changing the radio transmitter's default power level using the :c:func:`sdc_default_tx_power_set` function.
+  * Support for changing the peripheral latency mode using the :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set` function.
+  * Support for changing the default TX power using Kconfig options that start with ``CONFIG_BT_CTLR_TX_PWR_*``.
+
+For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
+
+Bluetooth mesh
+--------------
+
+* Added support for using :ref:`emds_readme`.
+  For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
+
+See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.
+
+Matter
+------
+
+* Introduced support for Matter over Wi-Fi to several samples.
+* Updated :ref:`ug_matter` with new pages about Matter SDK as well as information about Matter over Wi-Fi.
+* Removed the low-power configuration build type from all Matter samples.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``708685f4821df2aa0304f02db2773c429ad25eb8``.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+  * Support for Matter device factory data.
+    This includes a set of scripts for building the factory data partition content, and the ``FactoryDataProvider`` class for accessing this data.
+  * :ref:`Experimental support <software_maturity>` for Matter over Wi-Fi.
+
+Thread
+------
+
+* Added information about Synchronized Sleepy End Device (SSED) and SED vs SSED activity in the :ref:`thread_ot_device_types` documentation.
+* Removed multiprotocol support from :file:`overlay-cert.config` and moved it to :file:`overlay-multiprotocol.conf`.
+
+See `Thread samples`_ for the list of changes for the Thread samples.
+
+Zigbee
+------
+
+* Updated:
+
+  * The PAN ID conflict resolution is now enabled in applications that use the :ref:`lib_zigbee_application_utilities` library.
+    For details, see `Libraries for Zigbee`_.
+  * The default entropy source of Zigbee samples and unit tests to Cryptocell for SoCs that have Cryptocell.
+
+See `Zigbee samples`_ for the list of changes for the Zigbee samples.
+
```

Comment:
Please add a HomeKit section:

HomeKit
----------
* Added:

  * Support for Thread 1.3.

* Updated:
  * OTA DFU using the iOS Home app (over UARP - BLE and Thread) is no longer experimental.
  * Integrated HomeKit Accessory Development Kit v6.3 (ADK v6.3  e6e82026).
  * Improved current consumption for Thread Sleepy End Devices (SEDs) and Bluetooth LE peripherals.
  * Improved documentation.
 
* Fixed:

  * An issue where Bluetooth LE TX configuration was set to 0 dBm by default.
  * An issue where the Stateless Switch application crashed upon factory reset.

Reply:
Added.